app/assets/styles/components/ui-editor.css

```
@@ -12,99 +12,96 @@
     border-radius: 0.4rem;
     color: var(--mauve-12);
     font-size: 0.85rem;
     padding: 0.25em 0.3em;
   }
 
-  &:global(pre) {
+  & pre {
     background: var(--mauve-12);
     border-radius: 0.5rem;
     color: var(--mauve-1);
     font-family: 'JetBrainsMono', monospace;
```

Stylelint (17.4.0)

[error] Unexpected quotes around "JetBrainsMono" (font-family-name-quotes)

(font-family-name-quotes)